bigchaindb/log/setup.py

```diff
@@ -0,0 +1,154 @@
+"""Setup logging."""
+from copy import deepcopy
+import logging
+from logging.config import dictConfig
+import logging.handlers
+import pickle
+from socketserver import StreamRequestHandler, ThreadingTCPServer
+import struct
+import sys
+from multiprocessing import Process
+
+from .configs import PUBLISHER_LOGGING_CONFIG, SUBSCRIBER_LOGGING_CONFIG
+
+
+def setup_pub_logger():
+    dictConfig(PUBLISHER_LOGGING_CONFIG)
+    socket_handler = logging.handlers.SocketHandler(
```

Comment:
Just a note. Since we are logging to 'localhost' maybe it would be best to use a unix socket since there is less overhead. Also using udp instead of tcp would be more lightweight since there is no need to maintain connections or acknowledge packets, assuming that we don't care if we lose some messages, which shouldn't happen anyway since we are using `localhost`.